src/content/docs/nrql/nrql-tutorials/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -242,7 +243,7 @@ You might think that you may hit the limit by doing this. Try removing `EXTRAPOL
 
 When you include `EXTRAPOLATE` in a query, we calculate the ratio between the reported events and the total events. We then use this ratio to extrapolate an approximation of unsampled data. Keep in mind that only some queries support this use. When included in a NRQL query that doesn't support it or doesn't use sampled data, it has no effect.
 
-Note that homogenous data like throughput gets the most out of the `EXTRAPOLATE` function. It has less effect when attempting to extrapolate a count of distinct things (like `uniqueCount()` or `uniques()`). So, `EXTRAPOLATE` only works with NRQL queries that use one of the following aggregator functions:
+Remember that homogenous data like throughput gets the most out of the `EXTRAPOLATE` function. It has less effect when attempting to extrapolate a count of distinct things (like `uniqueCount()` or `uniques()`). So, `EXTRAPOLATE` only works with NRQL queries that use one of the following aggregator functions:
```

Comment:
I think you can drop "remember that" completely and jump right into "homogenous"

Reply:
It's good to drop filler words wherever you can, I'd say

Reply:
Agreed and changed and I will keep an eye out for this bad habit in the future, thank you for bringing attention to it